commitlint.config.js

```diff
@@ -1,27 +1,10 @@
 module.exports = {
   extends: ["@commitlint/config-conventional"],
-  plugins: ['commitlint-plugin-function-rules'],
   rules: {
     //   0 - Disabled, 1 - Warning, 2 - Error
     "body-max-line-length": [2, "always", 72],
-    "body-leading-blank": [1, "always"],
-    "header-max-length": [0], // level: disabled [2, "always", 72]
-    "function-rules/header-max-length": [
-      2, // level: error
-      'always',
-      (parsed) => {
-        if (parsed.scope === 'deps' || parsed.scope === 'dependencies') {
-          return [true];
-        }
-        if (parsed.header.length <= 72) {
-          return [true];
-        }
-        return [false, 'header must not be longer than 72 characters'];
-      },
-    ],
+    "header-max-length": [2, "always", 72],
```

Comment:
@arxeiss Does it mean that the renovate is disabled, or configured so it won't produce that long commit names?

Reply:
It is configured in a way, that it shouldn't produce long messages.

Message should be always `chore(deps): update everything`